terraform/aws/scenarios/omnibus-restore-backup/README.md

```diff
@@ -0,0 +1,6 @@
+# Omnibus Standalone Fresh Install
```

Comment:
Is this the name we want to call this scenario? fresh install and restore maybe? or you can have a fresh install without restoring a backup?

Reply:
Worth noting on the naming here is that I'll be removing the `omnibus-` prefix from all of the scenarios in a future refactoring commit as it is just a holdover from the previous implementation of testing.

Reply:
I copied this from another `README.md` (guess which one) - changed the title to `Omnibus Restore Existing Backup`